app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/CreateServiceRequestController.php

```diff
@@ -108,9 +109,12 @@ public function store(
         DB::beginTransaction();
 
         try {
+            $serviceRequestStatus = ServiceRequestStatus::orderBy('id', 'asc')->firstOrFail();
```

Comment:
Hey @kandarp-canyon, this doesn't look like what we discussed in the meeting earlier today.

We need to get the specific model here where the name is `New`, the classification is `Open`, and the `is_system_protected` value is `true`.

Reply:
Applied the conditions.